### PR TITLE
Adds partner sku in response data and a subscription active filter

### PR DESCRIPTION
**Pull request by rehan99000**
**Description:**
This PR adds the following:
- Partner sku for each subscription to help create a basket link
- A query param filter `filter_active` with values `true/false` to get only active subscriptions

**JIRA:**
https://edlyio.atlassian.net/browse/EDLY-1835

### CODE REVIEW

File: ecommerce/subscriptions/api/v2/serializers.py [hunk 1]
```diff
@@ -63,9 +64,15 @@ def get_subscription_status(self, product):
         """
         return product.attr.subscription_status
 
+    def get_partner_sku(self, product):
+        """
+        Get subscription's partner sku.
+        """
+        return product.stockrecords.first().partner_sku
```
Comment (iam-mhaseeb) [inline ecommerce/subscriptions/api/v2/serializers.py:71]: Can product have multiple stockrecords? If yes, will there partner_sku would be the same?

**Review by asadmanzoor93 — CHANGES_REQUESTED** [review]
Kindly update unit tests for the change.

File: ecommerce/subscriptions/api/v2/serializers.py (hunk 1 above)
Comment (rehan99000) [reply ecommerce/subscriptions/api/v2/serializers.py:71]: Nope.there will be a single stock record against each subscription

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Review by ziafazal — APPROVED** [review]
👍

**Review by iam-mhaseeb — APPROVED** [review]
If it's not urgent, adding tests suggested.

**Review by asadmanzoor93 — APPROVED** [review]

**Pull request merged**